cilium/cmd/encrypt_status.go

```
@@ -88,14 +91,40 @@ func getXfrmStats() (int, map[string]int) {
 func countUniqueIpsecKeys() int {
 	// trying to mimic set type data structure
 	// using void data type as struct{} because it does not use any memory
-	set := make(map[string]void)
+	keys := make(map[string]void)
 	xfrmStates, _ := netlink.XfrmStateList(0)
 	for _, v := range xfrmStates {
-		set[string(v.Crypt.Key)] = voidType
+		keys[string(v.Crypt.Key)] = voidType
 	}
-	return len(set)
+	return len(keys)
+}
+
+func maxSequenceNumber() string {
+	var maxSeqNum string
+	out, _ := exec.Command("ip", "xfrm", "state").Output()
```

Again, this might silently fail. Please, at least log the error if non-nil.